src/components/v5/shared/BurgerMenu/BurgerMenu.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { FC } from 'react';
 
-import Icon from '~shared/Icon/Icon';
+import Icon from '~shared/Icon';
 import { BurgerMenuProps } from './types';
 
-const displayName = 'v5.shared.BurgerMenu.BurgerMenu';
+const displayName = 'v5.shared.BurgerMenu';
 
-const BurgerMenu: FC<BurgerMenuProps> = ({ isVertical }) => (
+const BurgerMenu: FC<BurgerMenuProps> = ({ isVertical, setTriggerRef }) => (
   <button
     type="button"
```

Comment:
If it's just a icon inside the button, we should add aria-label to them

Reply:
aria-label added